inst/builtin/bs5/shiny/_variables.scss

```diff
@@ -70,8 +72,11 @@ $headings-font-weight: 400 !default;
 // Spacing
 $hr-margin-y: 2rem !default;
 
-// Buttons
+// Inputs
 $input-btn-font-size: 15px !default;
+$selectize-font-size: $input-btn-font-size !default;
```

Comment:
Please move this into `inst/bs3compat/_declarations.scss`

Reply:
I moved this into a new PR #763